docs/proposals/object-tracking.md

```diff
@@ -0,0 +1,152 @@
+## Motivation
+Object tracking is an important technology in the field of computer vision, which is widely used in security monitoring, intelligent transportation, etc. Generally, object tracking applications have high latency requirements, which cannot be met by cloud computing schemes. The object tracking schemes based on edge computing have the characteristics of low latency and data privacy security, and are the mainstream technology trend. 
+
+However, it is difficult for a single edge computing node to provide high-quality object tracking services. On the one hand, the coordination of multiple edge computing nodes is often required for across-camera object tracking. On the other hand, when there are large-scale tracking objects, a single computing node is difficult to handle. Multi-edge collaborative computing can accelerate large-scale object tracking applications and improve the tracking accuracy, which is the future development trend. 
+
+We propose the first open source end-to-end multi-edge collaborative object tracking solution. Based on KubeEdge's cloud-edge collaboration and resource management capabilities, we use multiple edge computing nodes to execute the AI inference tasks of object tracking in parallel. Our solution can not only reduce delay and improve throughput, but also will bring accuracy promotion.
+### Goals
+* Support single/multi-object tracking
+* Support across-camera object tracking
+* Support parallel object re-identification(ReID)
+* Support multi-camera data joint analysis and decision making
+
+
+## Proposal
+We propose using Kubernetes Custom Resource Definitions (CRDs) to describe the object tracking service specification/status and a controller to synchronize these updates between edge and cloud.
+![](./images/object-tracking-service-crd.png)
+
+### Use Cases
+
+* User can create typical multi-edge collaborative object tracking applications with providing AI models.
+
+## Design Details
+### CRD API Group and Version
+The `ObjectTrackingService` CRD will be namespace-scoped.
+The tables below summarize the group, kind and API version details for the CRD.
+
+* ObjectTrackingService
+
+| Field                 | Description             |
+|-----------------------|-------------------------|
+|Group                  | sedna.io     |
+|APIVersion             | v1alpha1                |
+|Kind                   | ObjectTrackingService   |
+
+
+### Object tracking service type definition
+
+[go source](/pkg/apis/sedna/v1alpha1/objecttrackingservice_types.go)
+
+#### Validation
+[Open API v3 Schema based validation](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#validation) can be used to guard against bad requests.
+Invalid values for fields (example string value for a boolean field etc) can be validated using this.
+
+Here is a list of validations we need to support :
+1. The `nodeSelector label` specified in the crd should exist in k8s.
```

Comment:
This is inaccurate since we need to support `kubectl label node` after objecttrackservice is created, we can't do nodeSelector validation.